cdc/sink/mysql_params.go

```diff
@@ -270,6 +270,14 @@ func generateDSNByParams(
 	dsnCfg.Params["writeTimeout"] = params.writeTimeout
 	dsnCfg.Params["timeout"] = params.dialTimeout
 
+	tidbPlacementMode, err := checkTiDBVariable(ctx, testDB, "tidb_placement_mode", "ignore")
+	if err != nil {
+		return "", err
+	}
+	if tidbPlacementMode != "" {
+		dsnCfg.Params["tidb_placement_mode"] = tidbPlacementMode
```

Comment:
```suggestion
		dsnCfg.Params["tidb_placement_mode"] = fmt.Sprintf(`"%s"`, tidbPlacementMode)
```

See the comments in test